README.md

```diff
@@ -20,38 +20,6 @@ The intention of this project is to create the easily configurable template, sum
 git clone https://github.com/Praqma/JenkinsAsCodeReference.git
 ```
 
-* Set proxy variables. You have to do it even if you are not using a proxy because there is [a bug in docker-compose](https://github.com/docker/compose/issues/3281) which makes args return None instead of empty string. Because of that, if you don't use proxy, you have to define empty environment variables http_proxy, https_proxy, no_proxy. Otherwise you would have those variables set to point out your proxy settings
```

Comment:
This part has to stay because it explains how to set proxy settings
What you can do is to say that people who are using docker-compose 1.7.0 should be aware of this issue